mla/threeml/sob_terms.py

```diff
@@ -55,47 +57,62 @@ def sob(self) -> np.ndarray:
 @dataclasses.dataclass
 class ThreeMLPSEnergyTermFactory(sob_terms.SoBTermFactory):
     """Docstring"""
+
     data_handler: data_handlers.ThreeMLDataHandler
     source: sources.PointSource
+    spectrum: spectral.BaseSpectrum = spectral.PowerLaw(1e3, 1e-14, -2)

```

Comment:
I think maybe we shouldn't have a default here. I don't think it really helps much to have it